app/controllers/site_controller.rb

```diff
@@ -1,4 +1,8 @@
 class SiteController < ApplicationController
   def index
   end
+
+  def search_gems
+  	@gems = GemsAdoption.search(params[:search])
+  end
```

Comment:
Actually according to how we planned this to work, this should go in `GemsAdoptionsController#index`. So when a user goes to this page we either show all the gems or the onces the search result returned.